concourse/pipelines/destroy-cloudfoundry.yml

```diff
@@ -296,7 +296,8 @@ jobs:
                 cp cf-tfstate/cf.tfstate updated-cf-tfstate/cf.tfstate
                 cd paas-cf/terraform/cloudfoundry || exit
```

Comment:
I think the `-chdir` is unnecessary because we're already in that directory due to this line.